libraries/ESP32/examples/Serial/Interrupt/Interrupt.ino

```diff
@@ -0,0 +1,24 @@
+// Serial, Serial1 and Serial2 are all of the type HardwareSerial.
+
+static void IRAM_ATTR onSerialRX(uint8_t c, void* user_arg){  
+	Serial.print(c);
```

Comment:
There is another thing that comes to mind. If `Serial::print` is not in IRAM, calls to it from IRAM interrupt will cause Flash to be read and then that will cause exception. In future Arduino ISRs will not be in IRAM, but currently they are.